packages/terra-application-utility/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 2.51.0 - (April 4, 2023)
```

Comment:
These headings are automatically added when we do a release; for now we can keep them under the `unreleased` section.

```suggestion
```